save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/CreateVulnerabilityView.kt

```diff
@@ -136,6 +137,7 @@ val createVulnerabilityView: VFC = VFC {
         "Base Score Calculator".t(),
         bodyBuilder = {
             cvssBaseScoreCalculator {
+                baseMetricsOfVulnerability = if (vulnerability.severity.isNotEmpty()) CvssVectorV3(vulnerability.severity.parsingVectorToMap()).baseMetrics else undefined
```

Comment:
maybe return 'null' instead of undefined?

Reply:
When i talked to @akuleshov7 about this, he suggested to return `undefined`. I can change this to `null` if necessary.

Reply:
`undefined` is js specific constantm but `null` is pure kotlin. I think better to use `null` when it's possible instead of `undefined`

Reply:
@akuleshov7 WDYT?

Reply:
he misunderstood me

Reply:
Done